content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -319,8 +319,8 @@ type: kubernetes.io/tls
 
 Referencing this secret in an Ingress tells the Ingress controller to
 secure the channel from the client to the load balancer using TLS. You need to make
-sure the TLS secret you created came from a certificate that contains a CN
-for `sslexample.foo.com`.
+sure the TLS secret you created came from a certificate that contains a Common
+Name (CN), aka Fully Qualified Domain Name (FQDN) for `sslexample.foo.com`.
```

Comment:
“aka” might count as [jargon](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-jargon-and-idioms); however, I'm inclined to accept this PR and leave room for future iteration if anyone has the time and inclination.

Reply:
I'm more fine with an extraneous commit (like what I'm about to make) than I am with leaving jargon in a doc.

Reply:
Thanks @sftim and @zacharysarah .  That's good to update jargon :)